### ISSUE TITLE
easy_localization_app.dart:244 Uncaught (in promise) TypeError: Cannot read property 'Symbol(dartx.split)' of null

### ISSUE THREAD

**Issue by mohammadne** [aissat/easy_localization#140]
error in  second line only in my flutter web ( not android and ios)
```
Locale _localeFromString(String val) {
  var localeList = val.split("_");
  return (localeList.length > 1)
      ? Locale(localeList.first, localeList.last)
      : Locale(localeList.first);
}
```

**Comment by Overman775** [comment]
@mohammadne show logs

**Comment by mohammadne** [comment]
this is resolved in 2.1.0+1
Thanks.
